Library/Formula/crosstool-ng.rb

```diff
@@ -42,7 +42,7 @@ def install
 
     args << "--with-grep=ggrep" if build.with? "grep"
 
-    args << "--with-make=gmake" if build.with? "make"
+    args << "--with-make=#{HOMEBREW_PREFIX}/bin/gmake" if build.with? "make"
```

Comment:
Rather than setting this hard on the `HOMEBREW_PREFIX` which will break if someone unlinks `gmake`, it may be worth using `#{Formula["make"].opt_bin}/gmake` instead.